pkg/exporter/exporter_test.go

```diff
@@ -120,6 +120,35 @@ type jsonEvent struct {
 
 const nodeName = "test-node-name"
 
+// countEvents counts the number of events and rate-limit-info messages in the given slice.
+// It returns (gotEvents, gotRateLimitInfo, gotDropped).
+func countEvents(eventsJSON []string) (int, int, uint64) {
+	gotEvents, gotRateLimitInfo, gotDropped := 0, 0, uint64(0)
```

Comment:
I'd used the named return parameters feature if you are anyway going to do it
```suggestion
func countEvents(eventsJSON []string) (gotEvents int, gotRateLimitInfo int, gotDropped uint64) {
```